notNeededPackages.json

```diff
@@ -3445,6 +3445,10 @@
             "libraryName": "json-rules-engine",
             "asOfVersion": "4.1.0"
         },
+        "json-schema-faker": {
+            "libraryName": "json-schema-faker",
+            "asOfVersion": "0.5.6"
```

Comment:
The types seem to have been there earlier than this version?

Reply:
Umm.. Just checked again. Seems type is bundled since `0.5.0`, but `@types/json-schema-faker` is 0.5.4 already. 😂

Reply:
Ah.